.travis.yml

```diff
@@ -11,29 +11,27 @@ matrix:
     - php: 7.3
     - php: 7.3
       env: deps=low
+    - php: 7.4
+    - php: 7.4
+      env: deps=low
     - php: nightly
       env: deps=low
     - php: nightly
   allow_failures:
     - php: nightly
 
-depth: 1
-
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
 
-before_script:
-    - if [[ $deps = low ]]; then composer update --prefer-lowest --prefer-stable; fi
+install:
+  - travis_retry composer install --no-interaction
+  - if [[ $deps = low ]]; then travis_retry composer update --prefer-lowest --prefer-stable --no-interaction; fi
```

Comment:
Here's how Laravel does it: https://github.com/laravel/framework/blob/v6.10.0/.travis.yml

Reply:
Thank you for the tip !